atoma-state/src/handlers.rs

```diff
@@ -766,6 +767,70 @@ pub(crate) async fn handle_state_manager_event(
     Ok(())
 }
 
+/// Handles an update to the number of compute units in a stack.
+///
+/// This function processes an update to the number of compute units in a stack by parsing the event data
+/// and updating the corresponding stack in the database.
```

Comment:
I think we should also mention that if the ratio is greater than the threshold, then we also submit a claim for funds for the stack, perhaps we should also rename the function given that potential branch as well